src/blocks/calendar.scss

```diff
@@ -0,0 +1,60 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: #000;
+}
+
+.calendar {
+  display: flex;
+  justify-content: center;
+  height: $calendar-height;
+  @for $day-number from 1 to $day-count {
+    &--month-length_#{$day-number} {
+      .calendar__day:nth-child(n + #{$day-number + 1}) {
```

Comment:
This is not a day-number
It is number-of-days!

Reply:
Agree ! a bit  longer name , more clear